src/cascadia/TerminalControl/Resources/en-US/Resources.resw

```diff
@@ -296,4 +296,8 @@ Please either install the missing font or choose another one.</value>
     <value>Select output</value>
     <comment>The tooltip for a button for selecting all of a command's output</comment>
   </data>
+  <data name="SessionRestoreMessage" xml:space="preserve">
+    <value>Restored</value>

```

Comment:
Do we need to put the args in here too? I'm not sure if there's any locales where the translation would require re-ordering those bits